.gitlab/package_build/linux.yml

```diff
@@ -80,6 +80,14 @@
   before_script:
     - export RELEASE_VERSION=$RELEASE_VERSION_7
 
+.agent_fips_build:
+  variables:
+    AGENT_MAJOR_VERSION: 7
```

Comment:
nit: I don't think you need to specify this, it's already implied by `.agent_7_build`

Reply:
removed in https://github.com/DataDog/datadog-agent/pull/31004/commits/9b56d4ede78dac2a6e16330fc01c157c61064699